app/views/candidate_mailer/_wait_or_respond.erb

```diff
@@ -0,0 +1,7 @@
+^You can wait to hear back about your other application(s) before making a decision. When the last decision is in, you’ll have 10 working days to respond.
```

Comment:
Stray `^`s in this file? I see they were in the original template but I don't think they do anything?

Reply:
Maybe retain the `.text` part of the file extension for consistency?